.gitignore

```diff
@@ -1,6 +1,10 @@
 .php_cs.cache
 docker-compose.override.yaml
 
+# IntelliJ project files:
```

Comment:
You should not put these files in projects gitignore but in a global gitignore on your system.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Ok !